src/test/common/services/craftApi.test.js

```diff
@@ -6,11 +6,15 @@ const chai = require('chai');
 const sinonChai = require('sinon-chai');
 const proxyquire = require('proxyquire');
 const { URL } = require('url');
+const nock = require('nock');
```

Comment:
crafts.test.js used to be in the test root folder, which used nock to mock HTTP calls. This is just as viable a means to test as the use of proxyquire and sinon mocks here, so they have been combined into a single relevant class.

The nock approach also makes testing errors easier and thus checks for the Promise reject path can be done.